core/models.py

```diff
@@ -37,8 +36,10 @@ class Meta:
 class Community(GenericModel):
     name = models.CharField(max_length=255)
     slug = models.SlugField(max_length=255, unique=True, blank=True)
-    author = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, related_name="authored_communities")
-    members = models.ManyToManyField(User, through="CommunityMember", related_name="communities")
+    author = models.ForeignKey(

```

Comment:
Revert this changes please. Seems your editor is set to shorter lines than project settings, you need to adjust. 
Our line lenght is 120 chars: https://github.com/pymasterspl/reddit/blob/6810df3c26114963e2be001cad26822025886179/pyproject.toml#L33

The problem: this change breakes git history and makes debugging harder also CR longer.